### ISSUE TITLE
Quick Pick placeholder and content text does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51831]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
ColorContrast,MAS1.4.3

**Repro Steps:**
1.	Launch VS Code.
2.	Navigate to Settings icon in Activity Bar->Press Enter->Select "File Icon Theme" from dropdown.
3.	Open color contrast analyzer and check the ratio.

**Actual:**
Color contrast ratio is less than 4.5:1 for the placeholder and the list items present on the screen

**Expected:**
Color Contrast ratio should not be less than 4.5:1.
Same issue observed on other color theme also.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

](url)

**User Impact:**
The user will not be able to see the controls.

**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
](url)
**Attachment for Reference:**

![accessibility a11y_vscode_fileicontheme_colorcontrast](https://user-images.githubusercontent.com/39204909/41394705-999e1b6c-6fc8-11e8-999c-2c5e554413e3.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This feels like law hanguing fruit. Assigning to July and also to @chrmarti to give code pointer to @misolori
